metadata-ingestion/src/datahub/ingestion/transformer/dataset_transformer.py

```diff
@@ -39,6 +46,55 @@ class DatasetTagsTransformer(DatasetTransformer, metaclass=ABCMeta):
     def aspect_name(self) -> str:
         return "globalTags"
 
+    @staticmethod
+    def merge_with_server_global_tags(
```

Comment:
Why don't you use patch? 
https://github.com/datahub-project/datahub/blob/a91c78cf314fe0abe7acbe68960fffaf66a8bf1c/metadata-ingestion/src/datahub/specific/dataset.py#L147

Reply:
I just extracted out what was already being used. Please see this https://github.com/datahub-project/datahub/pull/8351/files#diff-e8559426cc58f674e662afa59214574a4c140a785226579cad487eb65c2dc70eL39

Reply:
Doing that refactor across all transformers while really good is not really I want to do in this PR. That will increase the testing scope of this PR well outside of the current functionality that I am trying to add here.